### PR TITLE
Update api spec with changes for shared trips (MVP version)

### PR DESCRIPTION
**Pull request by safor**
https://app.shortcut.com/daytrip/story/40357/update-partner-api-spec

### CODE REVIEW

File: source/includes/trip-api.md [hunk 1]
```diff
@@ -790,7 +799,7 @@ Status code | Description
 
 ## Cancel endpoint
 
-This endpoint is used to cancel a booked trip. Only trips that have departure more than 24 hours in the future can be cancelled - this threshold is a subject to change.
+This endpoint is used to cancel a booked trip. For private trips only trips that have departure more than 24 hours in the future can be cancelled - this threshold is a subject to change. The shared trips can be cancelled at any time before departure, and compensated according to the selected [cancellation policy](#cancellationpolicytype).
```
Comment (kalaondr) [inline source/includes/trip-api.md:951]: This is problematic, current endpoint does not return information to the partner how much he needs to pay for the cancellation. Should be extended.

Comment (safor) [reply source/includes/trip-api.md:951]: Proposing to return `penalty` and `currency` (ae06991), which we already calculated in the code.

Comment (kalaondr) [reply source/includes/trip-api.md:951]: Looks good, will need to also update code for private trip logic to always return this. This API spec MR should be merged when the whole pool epic + this cancel change goes live.

Comment (safor) [reply source/includes/trip-api.md:951]: PR for changes in the endpoint implementation: https://github.com/mydaytrip/universal/pull/172

File: source/includes/trip-api.md [hunk 2]
```diff
@@ -755,6 +829,80 @@ curl -d '{ "optionId": "f0e34a1b-2b3d-4747-b426-292633b615b4", "pickUpAddressNot
...
+      "id":"519314e3-cd92-41e0-85b6-c652c345e9d0",
+      "type":"Private",
+      "englishSpeakingDriver": false,
+      "distanceKm":334,
+      "travelTimeMinutes":268,
+      "pickUp":{
+         "lat": 46.067648,
+         "lon": 7.775185,
+         "time":"2022-12-05T18:00:00Z",
+         "meetAndGreet": true
```
Comment (kalaondr) [inline source/includes/trip-api.md]: This should be false, see above.

**Review by kalaondr — CHANGES_REQUESTED** [review]

File: source/includes/trip-api.md (hunk 2 above)
Comment (safor) [reply source/includes/trip-api.md]: updated in a9f6f73544469625f3951b112800b772e495a9d4

**Review by kalaondr — APPROVED** [review]

**Pull request merged**